operations/research-and-development/engineering/team/README.md

```diff
@@ -61,3 +40,32 @@ Below is how SET prioritizes what is worked on.
    * Pick up any bugs in triage without an obvious owning feature team.
    * Work on bugs in this order: hotfix, release, customer reported, other.
 4. [~Ask R&D](https://community-daily.mattermost.com/core/channels/ask-r-and-d) channel — this channel should be monitored, but checking it once or twice per day is sufficient.
+
+## Sources of escalations
+
+Work for SET comes from various sources:
+
+1. The SET [Kanban JIRA board](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=33). Tickets in the TO DO column should be organized from highest priority to lowest priority, based on the [Priority of Work](./#priority-of-work) section. The SET lead should make sure that the tickets are priority ordered but team members can move tickets around to meet the correct priorities as necessary. If tickets get stuck in a certain state, it's also the SET responsibility to push them to completion, either by nudging people or taking them over.
+2. OpsGenie: Support has an e-mail address they will send urgent escalations to. These result in pages in OpsDuty, first sent to the SET Primary, then to SET Back-up and ultimately to the SET Lead when not timely acknowledged.
+3. The [~Sustained Engineering](https://community-daily.mattermost.com/core/channels/sustained-engineering) channel. Support may use this channel to ask questions or raise urgent issues.
+4. The [~Ask R&D](https://community-daily.mattermost.com/core/channels/ask-r-and-d) channel. This is the place where support and community can ask any type of R&D related questions.
+
+## Workflow
+When a new issue is raised, follow these steps:
+
+1. **Create a JIRA ticket** for the issue, initially connected to the "Sustained Engineering" team, and assigned to the SET Primary and moved to _In Progress_. Ensure the ticket describes the problem clearly and is as specific as possible. For example, if Mattermost is slow specify what is slow: the server, the client, a particular request.
+2. **Understand the customer impact and priority of the issue.** Determine the priority of the issue following [the levels specified here](https://support.mattermost.com/hc/en-us/articles/360038546151-Ticket-Priority-Guidelines), clearly mark this priority in the ticket. If the issue is really a feature request or low-urgency bug, change the ticket type and reassign it to the appropriate team. In that case, the SET responsibilities end. 
```

Comment:
again, this should be part of the playbook, we don't want people to forget doing this.

Reply:
```suggestion
2. **Understand the customer impact and priority of the issue.** Determine the priority of the issue following [the levels specified here](https://support.mattermost.com/hc/en-us/articles/4413076086548-Mattermost-Support-service-levels-and-their-severity), clearly mark this priority in the ticket. If the issue is really a feature request or low-urgency bug, change the ticket type and reassign it to the appropriate team. In that case, the SET responsibilities end. 
```

The link to the SLA descriptions changed.